apps/google-analytics-4/frontend/src/components/main-app/analytics-app/AnalyticsApp.styles.ts

```diff
@@ -39,4 +39,15 @@ export const styles = {
     width: '100px',
     margin: 'auto',
   }),
+  note: css({
+    overflow: 'hidden',
+    marginBottom: tokens.spacingM,
+  }),
+  noteContent: css({
+    textOverflow: 'ellipsis',
+    overflow: 'hidden',
+    display: '-webkit-box',
```

Comment:
I should know this but I'll just be honest and say I don't: do we need to provide fallbacks for non webkit browsers?
Like
```
display: -webkit-box;
display: -ms-flexbox;
display: flex;
```

Reply:
that is actually a great question! My previous understanding is that if the browser is not a webkit browser, it will default to `inherit` the display property instead. But I could be incorrect here! tricky to test as it seems like most iOS browsers use webkit - unless you know one one? will keep looking into it !

Reply:
Sounds good, I don't think this is really holding anything up then! Appreciate the thoughts